passes/techmap/abc9.cc

```diff
@@ -22,8 +22,13 @@
 // Berkeley Logic Synthesis and Verification Group, ABC: A System for Sequential Synthesis and Verification
 // http://www.eecs.berkeley.edu/~alanmi/abc/
 
-#define ABC_COMMAND_LUT "&st; &sweep; &scorr; "/*"&dc2; "*/"&retime; &dch -f; &ps -l; &if {W} -v; "/*"&mfs; "*/"&ps -l"
```

Comment:
Can we restore this as the default, and have &flow3 commented out (or under some even more experimental option?)
The original tries to emulate what `abc` did as much as possible

Reply:
Will do